### PR TITLE
Include Census claims for test_individual_response schema

### PR DESCRIPTION
**Pull request by MebinAbraham**
### What is the context of this PR?
- The `test_individual_response` schema requires the Census specific claims therefore, this PR sends those when launching the IR schema.
- Also fixes the Dockerfile python package issue cause the build to fail on master.

### How to review 
Ensure:
- existing features works as expected.
- test IR schema works with https://github.com/ONSdigital/eq-questionnaire-runner/pull/276

### CODE REVIEW

File: templates/launch.html [hunk 1]
```diff
@@ -151,41 +151,46 @@ <h3>Runner Data</h3>
...
+            </div>
+            <div class="field-container">
+                <label for="region_code">Region Code</label>
+                <input id="region_code" name="region_code" type="text" value="${regionCodeValue}" class="qa-region_code">
+            </div>
+            <div class="field-container">
+                <label for="channel">Channel</label>
+                <input id="channel" name="channel" type="text" value="RH" class="qa-channel">
+            </div>
+		`
```
Comment (JamesGardiner) [inline templates/launch.html:183]:
```suggestion
        `
```

File: templates/launch.html [hunk 2]
```diff
@@ -151,41 +151,46 @@ <h3>Runner Data</h3>
...
+     }
 
     function loadMetadata() {
         document.getElementById("submit-btn").disabled = true;
         document.getElementById("flush-btn").disabled = true;
 
-        let schema_name = document.getElementById("schema_name").value
-        if (schema_name.startsWith('census') || schema_name.startsWith('ccs')) {
-            includeCensusClaims()
+        const schema_name = document.getElementById("schema_name").value
```
Comment (JamesGardiner) [inline templates/launch.html:190]: This block could do with formatting

**Review by JamesGardiner — APPROVED** [review]
Tested with the `add-census-claims-to-ir-schema` image and LGTM. Some minor formatting issues

File: templates/launch.html (hunk 2 above)
Comment (MebinAbraham) [reply templates/launch.html:190]: Formatting the whole file muddied the PR, so I formatted the some JS without going overboard.

**Review by LJBabbage — APPROVED** [review]
Looks a lot more then what it is

**Pull request merged**